app/scripts/controllers/rif/rns/rns-delegate.js

```diff
@@ -43,4 +43,52 @@ export default class RnsDelegate {
   bindOperation (operation, member) {
     return nodeify(operation, member);
   }
+
+  /**
+   * Util method to invoke call transactions with web3
+   * @param contractInstance the contract instance to invoke
+   * @param methodName the contract method to invoke
+   * @param parameters the method parameters array
+   * @returns a Promise with the result of the transaction
+   */
+  call (contractInstance, methodName, parameters) {
```

Comment:
I like this! both call and sendTransaction, maybe i would call it just send (just to avoid confusion with sendTransaction of web3 since this is an abstraction)